From c57ffa33587da3d9610ba5885e9a15af695c5258 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sun, 1 Dec 2019 11:48:00 +0800
Subject: [PATCH] ARM: rockchip_defconfig: enable CONFIG_KERNEL_LZ4

LZ4 uncompress is faster than LZO. And same as arm64.
Save about 177 ms boot time on rk3126-bnd-d708-avb.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I59b008f8b6e36bd790310cace653c4e54800e438
---
 arch/arm/configs/rockchip_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 137b99480139..8c89dc87ee8a 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -1,5 +1,5 @@
 # CONFIG_LOCALVERSION_AUTO is not set
-CONFIG_KERNEL_LZO=y
+CONFIG_KERNEL_LZ4=y
 CONFIG_AUDIT=y
 # CONFIG_AUDITSYSCALL is not set
 CONFIG_NO_HZ=y
-- 
2.35.3

